src/shared/util.js

```diff
@@ -79,15 +79,31 @@ export function isPromise (val: any): boolean {
   )
 }
 
+export function getTrustedTypes() {
+  // TrustedTypes have been renamed to trustedTypes https://github.com/WICG/trusted-types/issues/177
+  return typeof window !== 'undefined' && (window.trustedTypes || window.TrustedTypes);
```

Comment:
might want to cache the result here since this will be called a lot (e.g. in `toString`)